front/app/containers/Admin/index.tsx

```diff
@@ -117,6 +119,9 @@ const AdminPage = memo<Props & WithRouterProps>(
     const [adminFullWidth, setAdminFullWidth] = useState(false);
     const [adminNoPadding, setAdminNoPadding] = useState(false);
 
+    const [contentBuilderLayoutVisible, setContentBuilderLayoutVisible] =
```

Comment:
This is the mechanism through which we are replacing the layout of the admin page.